addons/account/models/account_move.py

```diff
@@ -361,7 +361,7 @@ def _get_counterpart(self):
             counterpart = list(counterpart)[0:2] + ["..."]
         self.counterpart = ",".join(counterpart)
 
-    name = fields.Char(required=True, string="Label")
+    name = fields.Char(string="Label")
```

Comment:
I prefer it as required, IF you start to develop automations on account move lines creation without reference it is enought garbage now with generic names.

I mean, NAme required and improve the name generation in the automation, not remove the Termometer if you don't want to have fever.